templates/vhosts.j2

```diff
@@ -7,7 +7,9 @@ server {
     root {{ vhost.root }};
     {% endif %}
 
+    {% if vhost.no_index is defined and not vhost.no_index %}
```

Comment:
This `no_index` option would also need to be documented in the `defaults/main.yml